src/features/lessons/add-lesson/__tests__/add-lesson.test.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import { paths, useActions } from '@/shared';
+import { Router } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { AddLesson } from '@/features';
+import { commonHelpers } from '@/utils';
+import { groupsMock, mentorsMock, studentsMock, formData } from './mocks/mock-data.js';
+import { useStates, useStateMock } from './mocks/mock-useState';
+
+jest.mock('react-redux', () => ({ useSelector: jest.fn() }));
+
+jest.mock('@/shared/hooks', () => ({ useActions: jest.fn() }));
+
+describe('Render of EditLesson', () => {
+  let historyMock;
+  let mockStudentsSelector;
+  let mockGroupsSelector;
+  let mockMentorsSelector;
+  let mockEditLessonSelector;
```

Comment:
There should be mock**Add**LessonSelector (just add changes from the lesson) and everywhere, where it usage

Reply:
done)

Reply:
done)